deploy/chart/templates/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.name }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Values.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.name }}
+      annotations:
+        logz.io/skip: "true"
+        kubectl.kubernetes.io/default-container: manager
```

Comment:
Maybe it's worth adding in values `extraLabels` and `extraAnnotations` if users want to add their own labels/annotations.